extras/dev-docker/README.md

```diff
@@ -0,0 +1,17 @@
+# Docker image for GD2 development
+
+This docker image contains everything required to build and run GD2 during development.
+The image is available as a trusted build from Docker hub `gluster/glusterd2-dev`.
```

Comment:
The images don't exist yet, as I need this to be merged before I set up the builds on Docker hub